.github/workflows/reusable-docker-workflow.yml

```
@@ -0,0 +1,55 @@
+name: Reusable Docker Workflow
+
+on:
+  workflow_call:
+    inputs:
+      # Target
+      repository:
+        required: true
+        type: string
+      version-ref:
+        required: true
+        type: string
+      
+      # Build
+      build-cmd:
+        required: true
+        type: string
+      
+      # Reference image
+      reference-image:
+        required: true
+        type: string
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    steps:
+      # Clone repository whose reproducibility is to be tested
+      - name: Checkout repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          persist-credentials: false
+          ref: ${{ inputs.version-ref }}
+          repository: ${{ inputs.repository }}
+
+      # Install diffoci tool to help diff images
+      - name: Install diffoci
+        run: |
+          DIFFOCI_VERSION=v0.1.6
```

Minor but I think it's nicer to have this as an environment variable (maybe even defined at the top level of the workflow).


```suggestion
      - name: Install diffoci
        env:
          DIFFOCI_VERSION: v0.1.6
        run: |
```

This also makes it slightly easier to change the workflow in the future (e.g. make the `diffoci` version an input of the workflow).